core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -511,6 +513,32 @@ export class DotEditContentHtmlService {
         }
     }
 
+    private injectInlineBlockEditor(): void {
+        const doc = this.getEditPageDocument();
+        const editBlockEditorNodes = doc.querySelectorAll(
+            '.section-blog-post-content[data-content]'
```

Comment:
I don't think this selector will match all the cases of inline editing

Reply:
agree